canonical-websites/www.ubuntu.com
# Fix total search results

Fixes #2570

QA
--

First **CONNECT TO THE VPN**. Then:

``` bash
./run
```

Go to http://0.0.0.0:8001/search?q=cloud, see "returned 3170 results" (or similar).
